### PR TITLE
doc/rados: edit ops/monitoring.rst (1 of 3)

### PR DESCRIPTION
**Pull request by zdover23**
Line-edit the first third of doc/rados/operations/monitoring.rst.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/monitoring.rst [hunk 1]
```diff
@@ -13,11 +13,11 @@ Interactive mode
 ----------------
 
 To run the ``ceph`` tool in interactive mode, type ``ceph`` at the command line
-with no arguments.  For example:
+with no arguments. For example:
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:16]: I suspect that almost nobody ever does this

File: doc/rados/operations/monitoring.rst [hunk 2]
```diff
@@ -40,30 +41,32 @@ you may specify their locations:
 Checking a Cluster's Status
 ===========================
 
-After you start your cluster, and before you start reading and/or
-writing data, check your cluster's status first.
+After you start your cluster, and before you start reading and/or writing data,
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:44]: s/you start//

File: doc/rados/operations/monitoring.rst [hunk 3]
```diff
@@ -40,30 +41,32 @@ you may specify their locations:
...
 
 .. prompt:: ceph>
     :prompts: ceph>
-   
+
     status
 
-Ceph will print the cluster status. For example, a tiny Ceph demonstration
-cluster with one of each service may print the following:
+Ceph will print the cluster status. For example, a tiny Ceph "demonstration
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:67]: suggest:
 tiny Ceph sandbox cluster

File: doc/rados/operations/monitoring.rst [hunk 4]
```diff
@@ -86,31 +89,32 @@ cluster with one of each service may print the following:
...
-   Therefore, the amount of data actually stored typically exceeds the notional
-   amount stored, because Ceph creates replicas of the data and may also use 
-   storage capacity for cloning and snapshotting.
+The ``usage`` value reflects the *actual* amount of raw storage used. The ``xxx
+GB / xxx GB`` value means the amount available (the lesser number) of the
+overall storage capacity of the cluster. The notional number reflects the size
+of the stored data before it is replicated, cloned or snapshotted.  Therefore,
+the amount of data actually stored typically exceeds the notional amount
+stored, because Ceph creates replicas of the data and may also use storage
+capacity for cloning and snapshotting.
```
[the remark this answers is not in the document]
Comment (anthonyeleven) [reply doc/rados/operations/monitoring.rst:99]: I think this is just explaining the difference between raw and used, which admittedly might be rephrased into fewer and less-formal words.

File: doc/rados/operations/monitoring.rst [hunk 5]
```diff
@@ -84,33 +87,35 @@ cluster with one of each service may print the following:
...
 
 
 Watching a Cluster
 ==================
 
-In addition to local logging by each daemon, Ceph clusters maintain
-a *cluster log* that records high level events about the whole system.
-This is logged to disk on monitor servers (as ``/var/log/ceph/ceph.log`` by
-default), but can also be monitored via the command line.
+Each daemon in the Ceph cluster maintains a local log of events, and the Ceph
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:105]: s/local//

File: doc/rados/operations/monitoring.rst [hunk 6]
```diff
@@ -84,33 +87,35 @@ cluster with one of each service may print the following:
...
+To follow the cluster log, run the following command:
 
 .. prompt:: bash $
 
    ceph -w
 
-Ceph will print the status of the system, followed by each log message as it
-is emitted. For example:
+Ceph will print the status of the system, followed by each log message as it is
+emitted. For example:
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring.rst:118]: s/emitted/added/

**Review by anthonyeleven — APPROVED** [review]
You know the drill

File: doc/rados/operations/monitoring.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/operations/monitoring.rst:99]: unfinished business

**Comment by zdover23** [comment]
50822 - Reef backport
50823 - Quincy backport
50824 - Pacific backport

**Pull request merged**